web/src/PulseGuide.jsx

```
@@ -0,0 +1,104 @@
+  }, [isOpen]);
+
+  return (
+    <>
+      <button
+        className={`pulse-guide-fab ${isOpen ? 'open' : ''} ${isHovered ? 'hovered' : ''}`}
+        onClick={() => setIsOpen(!isOpen)}
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
+        aria-label="Pulse Guide Assistant"
+        title="Pulse Guide"
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)